client/extensions/woocommerce/style.scss

```diff
@@ -1,10 +1,64 @@
+.is-section-woocommerce {
+	.layout__primary {
+		display: flex;
+		justify-content: center;
+	}
+}
+
 .woocommerce {
+	flex-grow: 1;
+	max-width: 720px;
```

Comment:
>Do you know how the stats page over wrote the max-width? 

hmm... Stats page has no associated styles or classes yet.

>They should apply to all pages within the Store section.

The goal of Store stats page is to appear to the user as if they are in the same place as Site Stats, which has the `is-wide-layout` class, meaning `max-width: 1040px;`

If Stats is going to be the only page that needs a wide layout, then I can override these rules with higher specificity. I'm cool to 🚢